src/WebJobs.Extensions.Http/Config/HttpConfiguration.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Threading.Tasks.Dataflow;
+using Microsoft.Azure.WebJobs.Host.Config;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Http
+{
+    /// <summary>
+    /// Defines the configuration options for the Http binding.
+    /// </summary>
+    public class HttpConfiguration : IExtensionConfigProvider
```

Comment:
Yes, I think that is a good idea. The conflicts have been irritating - as you see in functions, I've had to namespace qualify in lots of places.

I'll come up with a new name